.github/workflows/e2e-rollup-workflow.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         echo "import answer from 'the-answer'; import _ from 'lodash';" | tee src/main.js
 
         yarn rollup -c
-        cat dist/bundle.js | grep -v "lodash"
+        ! cat dist/bundle.js | grep "lodash"
```

Comment:
TIL `!` applies to the whole pipe in Bash, not only the command that follows 🤔

Reply:
Yah I couldn't find any doc on this either. I just tried it. Hope that I haven't encounter any edge case 😅